packages/vc-verification/src/models/cache-interfaces.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  IRoleDefinitionV2,
+  RoleCredentialSubject,
+} from '@energyweb/credential-governance';
+import { VerifiableCredential } from '@ew-did-registry/credentials-interface';
+import { RoleEIP191JWT } from './credentials-types';
+import { IDIDDocument } from '@ew-did-registry/did-resolver-interface';
+
+/**
+ * An Interface to cache DID Document from Blockchain and SSI-Hub
```

Comment:
IMO, I would keep this comment generic as the interface isn't really tied to "Blockchain" or "SSI-HUB" in any way
```suggestion
 * An Interface to cache DID Documents
```